tests/ui/sanitizer/cfi-can-reveal-opaques.rs

```diff
@@ -0,0 +1,45 @@
+//@ needs-sanitizer-cfi
+//@ compile-flags: -Ccodegen-units=1 -Clto -Ctarget-feature=-crt-static -Zsanitizer=cfi --edition=2021
```

Comment:
Hey @rcvalle, I copied this from some other CFI UI test. What flags should I be putting here lol

Reply:
It looks good to me. The edition might not be necessary (unless you're using something that requires it?).

Reply:
Yep, unnecessary 👍